lib/cmds/space_cmds/alias_cmds/list.js

```diff
@@ -0,0 +1,61 @@
+
+import Table from 'cli-table3'
+
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+import { log } from '../../../utils/log'
+import paginate from '../../../utils/pagination'
+
+export const command = 'list'
+
+export const desc = 'List your space aliases'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space alias list')
+    .option('space-id', {
+      alias: 's',
+      describe: 'ID of the space the alias will belong to',
+      type: 'string'
+    })
+    .option('management-token', {
+      alias: 'mt',
+      describe: 'Contentful management API token',
+      type: 'string'
+    })
+    .epilog([
+      'Environment aliases is an [ALPHA] Feature',
+      'Copyright 2018 Contentful, this is a BETA release'
+    ].join('\n'))
+}
+
+export const aliases = ['ls']
+
+export async function environmentAliasList ({ context }) {
```

Comment:
do we need to also destructure the spaceId here if it's passed in by the user? 

(my understanding is that `context` is anything saved in the config, but not any options passed in when the user uses the command?)

Reply:
Thanks to middleware activeSpaceId will always haver the correct space ID whether it's passed or set in the config